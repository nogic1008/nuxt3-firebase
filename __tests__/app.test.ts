import { describe, expect, test } from '@jest/globals'
import { mount } from '@vue/test-utils'

import App from '~/app.vue'

describe('./app.vue', () => {
  test('should match snapshot', () => {
    const wrapper = mount(App)
    expect(wrapper.element).toMatchSnapshot()
  })
})
