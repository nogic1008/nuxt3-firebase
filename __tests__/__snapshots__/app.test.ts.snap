// Vitest Snapshot v1

exports[`./app.vue > should match snapshot 1`] = `
<div>
  Hello, World!
</div>
`;
